src/javascript/app/base/header.js

```diff
@@ -303,7 +303,7 @@ const Header = (() => {
 
             const validations = {
                 authenticate         : () => +get_account_status.prompt_client_to_authenticate,
-                cashier_locked       : () => /cashier_locked/.test(status),
+                cashier_locked       : () => /^cashier_locked$/.test(status),
```

Comment:
this doesn't work unless the only status in the array is `cashier_locked`. usually we will have multiple items in the status array.